modules/currency-l0/src/main/scala/org/tessellation/currency/http/routes/DebugRoutes.scala

```diff
@@ -0,0 +1,75 @@
+package org.tessellation.currency.http.routes
+
+import cats.effect.Async
+import cats.syntax.all._
+
+import org.tessellation.currency.modules.{Services, Storages}
+import org.tessellation.schema.cluster.SessionAlreadyExists
+import org.tessellation.schema.node.InvalidNodeStateTransition
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.ext.http4s.SnapshotOrdinalVar
+import org.tessellation.sdk.infrastructure.consensus.declaration.PeerDeclaration
+import org.tessellation.sdk.infrastructure.consensus.{ConsensusResources, PeerDeclarations}
+
+import derevo.circe.magnolia.{decoder, encoder}
+import derevo.derive
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class DebugRoutes[F[_]: Async](
```

Comment:
I will extract these. I just need to pass all the services manually instead of `Services` fascade.